CodeGen/README.md

```diff
@@ -31,3 +31,9 @@ CodeGen.exe --ver
 
 Hit TAB and it should now suggest `--version` and `--verbose` parameters.
 This should work with any .exe that is compiled with Dragonfruit's app model.
+
+## nanoFramework project generation
```

Comment:
Maybe just call this `## nanoFramework`.
We should also mention this supported framework in the `Build Targets` section at the top of the readme, with a link to this section with more information.